ndmg/stats/qa_tensor.py

```diff
@@ -111,55 +118,42 @@ def plot_directions(peak_dirs,peak_values,x_angle,y_angle,fname,size=(300,300)):
     scene.pitch(y_angle)
     
     window.show(scene, size = size)
-    
-def pad_im(image,max_dim,pad_val):
-    """
-    Pads an image to be same dimensions as given max_dim
-    
-    Parameters
-    -----------
-    image: 3-d RGB np array of image slice
-    max_dim: dimension to pad up to
-    pad_val: value to pad with
-    
-    Returns
-    -----------
-    padded_image: 3-d RGB np array of image slice with padding
-    """
-    #pad only in first two dimensions not in rgb
-    pad_width = (((max_dim-image.shape[0])//2,(max_dim-image.shape[0])//2),((max_dim-image.shape[1])//2,(max_dim-image.shape[1])//2),(0,0))
-    padded_image = np.pad(image, pad_width=pad_width, mode='constant', constant_values=pad_val)
-    
-    return padded_image
 
 def create_qa_figure(peak_dirs,peak_values,output_dir,model):
     """
     Creates a 9x9 figure of the 3-d volume and saves it
     
     Parameters
     -----------
-    peak_dirs: np array of peak_dirs 
-    peak_values: np array of peak_values
-    output_dir: location to save qa figure
-    model: model type (CSA, CSD)
+    peak_dirs: np array
+        peak_dirs 
+    peak_values: np array
+        peak_values
+    output_dir: str
+        location to save qa figure
+    model: str
+        model type (CSA, CSD)
```

Comment:
I'm still not happy with these docstrings.
Give `peak_dirs`, `peak_values`, and `model` descriptions that say what they actually are